### ISSUE TITLE
transaction not finished when no media files

### ISSUE THREAD

**Issue by mat-m** [michitux/dokuwiki-plugin-move#7]
Hello

I move a namespace, asking for media and pages. If there is no media, the transaction is considered not finished and I will get the Continue/abort buttons on the next trip to the plugin page. But transaction is actually finished.

Could someone confirm this or is it related to my config ?

**Comment by michitux** [comment]
We have released a massively improved and changed version today, please check if this issue still exists.

**Comment by mat-m** [comment]
Just tested it:
i got the following error:
```
Warning: require_once(__DIR__/handler.php): failed to open stream: No such file or directory in /home/data/www/wikippd/lib/plugins/move/helper/rewrite.php on line 14 
Fatal error: require_once(): Failed opening required '__DIR__/handler.php' (include_path='.:/usr/share/php5:/usr/share/php5/PEAR') in /home/data/www/wikippd/lib/plugins/move/helper/rewrite.php on line 14
```

/home/data/www/wikippd/lib/plugins/move/helper/handler;php is here.

**Comment by dam09fr** [comment]
I got the same error on "handler.php" include !
I need to edit "helper/rewrite.php" and change this to get the plugin working:
- require_once(__DIR__ . '/handler.php');
+ require_once(dirname(__FILE__) . '/handler.php'); 
My server is still running the old (and obsolete) PHP 5.2 so __DIR__ is not available (added in PHP 5.3).

**Comment by mat-m** [comment]
@dam09fr : Thanks, it also did the trick for me (same issue, actually)

@michitux : Issue not there anymore with the new plugin version, thanks !
